test/unit/metadata.js

```diff
@@ -772,4 +774,67 @@ describe('metadata', function () {
       should(upToDate(doc).errors).be.undefined()
     })
   })
+
+  describe('generated a doc with conflict', () => {
+    it('should get the correct path', () => {
```

Comment:
Maybe we don't need to start every test description with *should*?

Reply:
This is recommended by `mocha`: https://mochajs.org/#getting-started

Reply:
I would still prefer sticking to the current convention:

```bash
$ git co master 
$ grep -r "it('should" test/ | wc -l
1
$ grep -r "it('[^s]" test/ | wc -l
591
```